src/geofabrics/dem.py

```diff
@@ -104,14 +105,14 @@ def _extract_points(self):
         self._points['Z'][len(land_x):] = foreshore_z
 
     @property
-    def points(self):
+    def points(self) -> numpy.ndarray:
         """ The reference DEM points after any extent or foreshore value
         filtering. """
 
         return self._points
 
     @property
-    def extents(self):
+    def extents(self) -> geopandas.GeoDataFrame:
         """ The extents for the reference DEM """
```

Comment:
Can you clarify this term in comments or readme somewhere? (If it is not already done  :-) )
What type of info it is? (cache matrix or contours? holes included or just regarding boundaries? )
It is for each tile or for the full domain?

Reply:
Good idea. Adding a description now

Reply:
Added and also in the wiki. Hopefully clearer now.